### PR TITLE
swapped position of donate and successes on nav bar

### PR DESCRIPTION
**Pull request by RaspberryRat**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -244,6 +244,8 @@ GEM
       net-protocol
       timeout
```
Comment (kasugaijin) [inline Gemfile.lock:245]: no need to commit `Gemfile.lock` on PRs

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -9,36 +9,36 @@
```
Comment (kasugaijin) [inline app/views/layouts/application.html.erb:9]: looks like a linter ran here and made more changes in addition to the actual required code changes, please make a PR without auto linting

**Review by kasugaijin — COMMENTED** [review]
In addition to the above comments, generally it's good practice to include a screenshot of the changes if they are UI/front end changes in the PR...and to include a short, succint description of what the PR does.

**Comment by RaspberryRat** [comment]
PR had Gemfile and autolinting on save. Closing and submitted as a new PR.

**Pull request closed without merging**